tests/pr_test.go

```diff
@@ -0,0 +1,96 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/cloudinfo"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+const resourceGroup = "geretain-test-transit-gw"
```

Comment:
Did you manually create this group? I don't see it listed in https://github.ibm.com/GoldenEye/ge-dev-account-management/blob/master/main.tf
Can you add it into that repo so its managed by terraform just like all our other groups?

Reply:
@ocofaigh - Yes, it's been created, that's why the PR tests are passing

Reply:
It was manually created though, so lets make sure we add it to that terraform repo, as we manage all groups using terraform